packages/target-tauri/src-tauri/src/stickers.rs

```diff
@@ -0,0 +1,112 @@
+use anyhow::Context;
+use log::error;
+use percent_encoding::percent_decode_str;
+use tauri::{Manager, UriSchemeContext, UriSchemeResponder};
+use tokio::fs;
+
+use crate::state::deltachat::DeltaChatAppState;
+
+pub(crate) fn delta_stickers_protocol<R: tauri::Runtime>(
+    ctx: UriSchemeContext<'_, R>,
+    request: http::Request<Vec<u8>>,
+    responder: UriSchemeResponder,
+) {
+    // info!("dcsticker {}", request.uri());
+
+    // URI format is dcsticker://<account folder name>/<sticker pack>/<sticker filename>
```

Comment:
But this is different for windows right? So either remove comment or update it with windows info.

Reply:
```suggestion
    // URI format is dcsticker://<account folder name>/<sticker pack>/<sticker filename>
    // - Mac, linux, iOS: dcsticker://<account folder name>/<sticker pack>/<sticker filename>
    // - windows, android: http://dcsticker.localhost/<account folder name>/<sticker pack>/<sticker filename>
```